tox.ini

```diff
@@ -46,7 +46,7 @@ deps =
   codespell
 
 commands =
-  codespell
+  codespell -L crate
```

Comment:
Can you add to https://github.com/aws-observability/aws-otel-python-instrumentation/blob/main/.codespellrc

Reply:
Didn't even realize we already had a file for this! :D 
Cool.

Reply:
Done